src/context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const AppContext = React.createContext();
```

Comment:
Redundant `const` declaration. Can be simplified to `export default React.createContext()`